invenio_communities/assets/semantic-ui/js/invenio_communities/community/featured-communities/FeaturedCommunity.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from "react";
+import { Grid, Segment, Header } from "semantic-ui-react";
+import PropTypes from "prop-types";
+import { Image } from "react-invenio-forms";
+
+export default class FeaturedCommunity extends Component {
+  render() {
+    const {
+      computerColumnWidth,
+      mobileColumnWidth,
+      tabletColumnWidth,
+      widescreenColumnWidth,
+      community,
+    } = this.props;
+    return (
+      <Grid.Column
+        mobile={mobileColumnWidth}
+        tablet={tabletColumnWidth}
+        widescreen={widescreenColumnWidth}
+        computer={computerColumnWidth}
+        textAlign="center"
+      >
+        <Segment compact className="m-auto">
+          <a href={community.links.self_html}>
```

Comment:
Is the link `self_html` going to be there for sure? It is not validated anywhere, `propTypes` only validates that `community` is an object

Reply:
Yes, `self_html` is always present